components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -202,12 +203,13 @@ func (ir *ideRoutes) HandleCreateKeyRoute(route *mux.Route, hostKeyList []ssh.Si
 			} `json:"hostKey"`
 		}{}
 
-		_, pvk, err := ed25519.GenerateKey(crand.Reader)
+		privateKey, err := ecdsa.GenerateKey(elliptic.P256(), crand.Reader)
```

Comment:
It sounds like there are 4 scenarios we must do, to help prevent regressions with SSH:
1. connect with vs code desktop
2. connect with Jetbrains
3. port-forward an app from a workspace from either IDE
4. use a branch version of an extension from https://github.com/gitpod-io/gitpod-vscode-desktop extension, which depends on `dev-tunnel-ssh` (I am not sure which extension is intended to be tested or how to use it)

@iQQBot which test scenarios are covered by automated tests now for different IDEs?

@jeanp413, can we ask for your help reviewing/testing? I ask for two reasons. You did testing last time for JetBrains, and, it would be good to get input from Exp team on test scenarios due to IDE knowledge needed.

Reply:
@jeanp413 I can do, working with @iQQBot now on setup, I installed gateway and setup to force ssh over http tunnel